paper/basic_training.tex

```diff
@@ -730,11 +731,170 @@ \subsubsection{Summary}
+be classified into three main categories: volume rescaling, weakly
+coupled, and extended ensemble barostats \cite{ShellNotes,
+tuckermanBook}. The next section will describe the main differences
+between these barostats, and give some recommendations for proper
+use.
+
+
+\subsubsection{Popular and Notable Barostats}
+Within this section, a few notable barostats will be introduced to the
+reader, describing a high-level summary of each, with some of the issues
+associated with them as well. This is not an exhaustive list of
+barostats and barostat algorithms, just a sampling of popular and
+historic ones used in MD.
+
+\subparagraph{Volume Rescaling}
+\begin{enumerate}[listparindent=\parindent]
+    \item \textbf{Volume Rescaling}
+
+Volume rescaling barostats are the simplest example of pressure
+control in molecular simulations. Every time this barostat is
+executed, the volume of the system is modified to produce the exact
+pressure desired. This does \textbf{not} sample the proper
+ensemble, this cannot be used for production
+sampling \cite{ShellNotes}. This also does not smoothly approach the
+target pressure either, which might cause very unphysical issues
+with the system during time integration and force calculation.
+
+\end{enumerate}
+
+\subparagraph{Weakly Coupled Barostats}
+\begin{enumerate}[listparindent=\parindent]
+\item \textbf{Berendsen}
+
+The Berendesen \cite{berendsen1984molecular} barostat is very
+similar to the Berendsen thermostat discussed earlier. With the
+goal of improving upon the volume rescaling methods mentioned above.
+This was to be achieved by coupling the system to a weakly
+interacting pressure bath \cite{berendsen1984molecular}. This bath
+would scale the volume periodically by a scaling factor, but would
+also produce more realisitc fluctuations in the pressure as it
+slowly approached the target pressure. In contrast to volume
+rescaling, Berendsen will approach the target pressure more
+realistically, but the ensemble it is sampling from is not well
+defined and cannot be guaranteed to be NPT or NPH. Berendsen can
+be useful for the beginning stages of equilibration, but should
+\textbf{not} be used for production sampling.
+
+\end{enumerate}
+
+\subparagraph{Extended System Barostats}
+\begin{enumerate}[listparindent=\parindent]
+\item \textbf{Andersen Barostat}
+
+First described by Andersen \cite{andersen1980molecular} in 1980, the
+system is coupled to a ficticious pressure bath, by adding an additional
+degree of freedom to the equations of motion. This behaves as if the
+system is being acted upon by an isotropic piston. This is similar to
+the Nos\'{e}-Hoover thermostat, which is also an extended system
+algorithm. This barostat does sample the correct ensemble, however,
+it is isotropic in nature and applying anisotropic pressures to
+parts of the system is not possible.
+
+\item \textbf{Parrinello-Rahman Barostat}
+
+The Parrinello-Rahman \cite{Parrinello1981} barostat was an extension
```

Comment:
Probably avoid "was"; used here it sounds like you're saying this barostat is no longer used (it "was" an extension but no longer is?). I'd say "The ... barostat is an extension of..."